src/RestEaseUnitTests/ImplementationBuilderTests/RawQueryParameterTests.cs

```diff
@@ -79,26 +74,12 @@ public void AddsTwoRawQueryParam()
                 .Callback((IRequestInfo r) => requestInfo = r)
                 .Returns(Task.FromResult(false));
 
-            implementation.FooAsync("&test=test2", "&test3=test4");
-
-            Assert.NotNull(requestInfo.RawQueryParameters);
-            Assert.Equal("&test=test2&test3=test4", requestInfo.RawQueryParameters.SerializeToString(null));
-        }
-
-        [Fact]
-        public void AddsThreeRawQueryParam()
-        {
-            var implementation = this.builder.CreateImplementation<IThreeRawQueryStrings>(this.requester.Object);
-            IRequestInfo requestInfo = null;
-
-            this.requester.Setup(x => x.RequestVoidAsync(It.IsAny<IRequestInfo>()))
-                .Callback((IRequestInfo r) => requestInfo = r)
-                .Returns(Task.FromResult(false));
-
-            implementation.FooAsync("&test=test2", "&test3=test4", "&test5=test6");
+            implementation.FooAsync("test=test2", "test3=test4");
 
             Assert.NotNull(requestInfo.RawQueryParameters);
-            Assert.Equal("&test=test2&test3=test4&test5=test6", requestInfo.RawQueryParameters.SerializeToString(null));
+            Assert.Equal(2, requestInfo.RawQueryParameters.Count());
+            Assert.Equal("test=test2", requestInfo.RawQueryParameters.First().SerializeToString(null));
+            Assert.Equal("test3=test4", requestInfo.RawQueryParameters.Skip(1).Take(1).First().SerializeToString(null));
```

Comment:
`ElementAt(1)` is a shorthand for this, although the rest of the tests do `ToList()`. This also lets you assert that the number of elements in `RawQueryParameters` is correct